website/content/en/docs/amazon-sagemaker-integration/sagemaker-components-for-kubeflow-pipelines.md

```diff
@@ -10,10 +10,91 @@ These components integrate SageMaker with the portability and orchestration of K
 
 ## Available components
 
-You can create a Kubeflow Pipeline built entirely using SageMaker components, or integrate individual components into your workflow as needed. Available Amazon SageMaker components can be found in the [Kubeflow GitHub repository](https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker).
+You can create a Kubeflow Pipeline built entirely using SageMaker components, or integrate individual components into your workflow as needed. Available Amazon SageMaker components can be found in the [Kubeflow Pipelines GitHub repository](https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker).
 
-## Tutorials
+Currently, we support two versions of these SageMaker components - the traditional v1 components and the ACK Based v2 components. The Sample included with Kubeflow is currently based off of the v2 Training Component. However, you can just as easily configure and use all others by referring to our more detailed guides over at the pipelines repo - 
```

Comment:
There are two versions of SageMaker components - boto3 based v1 components and SageMaker Operator for K8s (ACK) based v2 components.